README.md

```diff
@@ -1,55 +1,20 @@
-gokeepasslib
+gokeepasslib (unstable)
```

Comment:
I assume it is stable now.

Do you want to keep these changes to the Pull Request?

Reply:
Sure why not, just make a new README like the original one :)

Reply:
Very well